docs/docs/outputs/output-formats.md

```diff
@@ -81,3 +68,64 @@ When authoring a Go template the data source is Tracee's `trace.Event` struct, w
 Go template can utilize helper functions from [Sprig](http://masterminds.github.io/sprig/).
 
 For example templates, see [tracee/cmd/tracee-rules/templates](https://github.com/aquasecurity/tracee/tree/main/cmd/tracee-rules/templates).
+
+The following sections can be specified as part of go templates:
+
+```
+output:
+    # gotemplate:
+    #     template: /path/to/my_template1.tmpl
+    #     files:
+    #         - /path/to/output1.out
+    #         - /path/to/output2.out
+```
+
+### Webhook
```

Comment:
Should Webhook be placed in the output options file?

Reply:
it's under output hierarchy in the yaml right?

Reply:
yes